### PR TITLE
add Centos Stream 9 (stream9) userenv

### PR DESCRIPTION
**Pull request by k-rister**

### CODE REVIEW

File: userenvs/stream9.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+  "workshop": {
+    "schema": {
+      "version": "2020.03.02"
+    }
+  },
+  "userenv": {
+    "name": "stream8",
```
Comment (HughNhan) [inline userenvs/stream9.json:8]: Should it be stream9?

Comment (k-rister) [reply userenvs/stream9.json:8]: It should, editing fail on my part.  Nice catch!

File: userenvs/stream9/userenv.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+  "userenv": {
+    "name": "stream8",
```
[the remark this answers is not in the document]
Comment (k-rister) [reply userenvs/stream9/userenv.json]: FYI, this file is actually used to generate the other file you commented on -- so this file is where the actual problem lies.

**Comment by k-rister** [comment]
> ship it!

lol...its kinda funny that you already reviewed it, but I'm glad you did since you got my mistake.  I honestly have no idea if this is correct and I was going to let CI decide if it so it will be very interesting to see what thee results of the CI tests are (this is why I put the PR in draft status).

**Pull request closed without merging**